CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Fix an issue with checkbox list items separated with blank lines (#602).
 * Require package `web: '>=0.4.2 <2.0.0'`.
 * Fix several `RangeError` hazards in links (#623).
+* Export 'lintk_reference_definition_syntax.dart' publicly (#626).
```

Comment:
Nit: that's an implementation detail. This should mention the new *members* that are now visible, right?

Reply:
Done!